docs/azure/aksextensions.md

```diff
@@ -0,0 +1,129 @@
+---
+Order: 7
+Area: azure
+TOCTitle: AKS Extensions
+ContentId: 131f9633-5446-4384-96ca-7bff2e3dc0fc
+PageTitle: Working with AKS tools and diagnostics in Visual Studio Code
+DateApproved: 5/03/2022
+MetaDescription: Working with AKS tools and diagnostics in Visual Studio Code
+---
+# Working with AKS tools and diagnostics in VS Code
+
+This document will walk you through some of the ways you can interact with your AKS clusters directly from Visual Studio Code. [Azure Kubernetes Services](https://azure.microsoft.com/en-in/services/kubernetes-service/#overview) is a fully managed [Kubernetes](https://azure.microsoft.com/en-in/topic/what-is-kubernetes/#overview) service. Azure Kubernetes Service (AKS) offers serverless Kubernetes, an integrated continuous integration and continuous delivery (CI/CD) experience and enterprise-grade security and governance, is an open-source system for automating deployment, scaling, and management of containerized applications. We will show you how to run [diagnostic health-checks](https://docs.microsoft.com/en-gb/azure/aks/concepts-diagnostics) on your AKS cluster, launch [AKS Periscope](https://github.com/azure/aks-periscope) for more in-depth troubleshooting, deploy [Azure Service Operator](https://github.com/Azure/azure-service-operator), or generate [GitHub Actions Starter Workflows](https://github.com/actions/starter-workflows).
```

Comment:
Can you remove the locale section of the URLs so that it will resolve to the readers locale.
I see 'en-in', 'en-gb', 'en-us' in a lot of the azure.microsoft.com and docs.microsoft.com URLs.